migrations/V20190830093000__create_fraud_schema.sql

```diff
@@ -0,0 +1,64 @@
+CREATE SCHEMA fraud AUTHORIZATION event_system_owner;
+
+CREATE TABLE fraud.upload_session
+(
+    id	                    BIGINT NOT NULL,
```

Comment:
Should our `id` columns be automatically-generated? (`SERIAL` or `GENERATED ALWAYS AS IDENTITY` I believe?)

Reply:
You're right ... I meant to go back and do that ... wasn't sure of the Postgres-ism for it at the time.

Reply:
Commit pushed